.github/workflows/ci-win.yml

```diff
@@ -12,20 +11,23 @@ jobs:
         node: [18, 20, 22]
 
     steps:
-    - uses: actions/checkout@v4
-    - name: Use Node.js ${{ matrix.node }}
-      uses: actions/setup-node@v4
-      with:
-        node-version: ${{ matrix.node }}
-    - name: Installing project dependencies
-      run: |
-        npm ci
-    - name: Lint
-      run: |
-        npm run lint
-    - name: Test
-      run: |
-        npm test
-    - name: Build
-      run: |
-        npm run docs:build
\ No newline at end of file
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Installing project dependencies
+        run: |
+          npm ci
+      - name: Format
```

Comment:
I think we probably only need to add the formatting check to the main _ci.yml_ job.  Probably the same thing for the lint task.

Just a thought to save a little time / compute from redundant work.